pkg/chunked/storage_linux.go

```diff
@@ -1280,6 +1280,14 @@ func typeToOsMode(typ string) (os.FileMode, error) {
 	return 0, fmt.Errorf("unknown file type %q", typ)
 }
 
+func isRootDirectory(path string) bool {
+	path = filepath.Clean(path)
+	for strings.HasPrefix(path, "../") {
```

Comment:
Why not 

path=strings.Prefix(s, ".//")

Do we care about "./"?

Reply:
'/' is automatically removed by `filepath.Clean()` AFAICS, I've added a new test case to validate a sequence of `./`.

yes `strings.CutPrefix` is better.  Changed and pushed a new version